services-js/commissions-app/src/client/common/TextInput.tsx

```diff
@@ -25,12 +27,19 @@ export default class FormWithElement extends React.Component<Props> {
         </label>
         <input
           name={this.props.name}
-          className="txt-f txt-f--sm"
+          className={
+            this.props.error ? 'txt-f txt-f--sm txt-f--err' : 'txt-f txt-f--sm'
+          }
           placeholder={this.props.placeholder}
           id="FeedbackForm-${this.props.name}"
           type="text"
           required={this.props.required}
         />
+        {this.props.error && (
+          <div className="t--subinfo t--err m-t100">
+            Please enter an email address
```

Comment:
This line makes the error message always be "Please enter an email address" no matter what error is passed in through props.

Change this to render the `this.props.error` string instead of this hard-coded value.

Reply:
The way to test it is to change the `error` you’re passing in in the story to a different string. When Storybook shows that string, rather than "Please enter an email address" then you’ve got it.